DigitalLearningSolutions.Web.Tests/ViewModels/MyAccount/MyAccountViewModelTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.MyAccount
+{
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.MyAccount;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class MyAccountViewModelTests
+    {
+        [Test]
+        public void MyAccountViewModel_AdminUser_and_DelegateUser_populates_expected_values()
+        {
+            // Given
+            var adminUser = MyAccountHelper.GetDefaultAdminUser();
+            var delegateUser = MyAccountHelper.GetDefaultDelegateUser();
+            var centreName = MyAccountHelper.GetDefaultCentreName();
+            
+            // When
+            var returnedModel = new MyAccountViewModel(adminUser, delegateUser, centreName);
+
+            // Then
+            returnedModel.FirstName.Should().BeEquivalentTo(adminUser.FirstName);
+            returnedModel.Centre.Should().BeEquivalentTo(centreName);
+            returnedModel.Surname.Should().BeEquivalentTo(adminUser.LastName);
+            returnedModel.ProfilePicture.Should().BeEquivalentTo(adminUser.ProfileImage);
+            returnedModel.DelegateNumber.Should().BeEquivalentTo(delegateUser.CandidateNumber);
+            returnedModel.User.Should().BeEquivalentTo(adminUser.EmailAddress);
```

Comment:
There are a lot of assertions here to test each individual property. Do we think an Assert.Multiple() might be of benefit? https://docs.nunit.org/articles/nunit/writing-tests/assertions/multiple-asserts.html
The docs suggest it would allow multiple errors to be displayed at once, rather than terminating on the first, but I don't know if that's how the present code already works - or as they are testing what should be the same model if one property is wrong we can assume the rest will be wrong as well.

Reply:
Looked into this, apparently FluentAssertions don't work with the NUnit Assert.Multiple, but it has a similar AssertionScope that I'll use instead.